pkgs/development/python-modules/pytmx/default.nix

```
@@ -10,16 +10,18 @@
 
 buildPythonPackage rec {
   pname = "pytmx";
-  version = "3.31";
+  version = "3.32";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `nativeBuildInputs`

---

Using `setuptools-scm` instead of `setuptools` worked. Thanks.